docs/src/pages/components/data-grid/columns/columns.md

```diff
@@ -68,6 +68,8 @@ To prevent the resizing of a column, set `resizable: false` in the `ColDef`.
 - https://www.jqwidgets.com/react/react-grid/#https://www.jqwidgets.com/react/react-grid/react-grid-columnsresize.htm
 -->
 
+Alternatively, to disable all columns resize, set the prop `disableColumnResize` to `true`.
```

Comment:
I think that we should explain how to enable column resizing (by using XGrid) as most will come from DataGrid.